app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -197,6 +198,23 @@ define(function (require, exports, module) {
       this.notifier.triggerAll(Notifier.PROFILE_CHANGE, {
         uid: uid
       });
+    },
+
+    _supportsAvatarUpload () {
```

Comment:
Since this function is called by consuming views, can you expose it as a public function, and add a JS doc?